src/Components/BrowseAll.jsx

```diff
@@ -18,9 +19,8 @@ export default function BrowseAll() {
       <h1>Borwse ALL</h1>
       <main className="card-container">
         {allArticles.map((article, index) => {
-          return (
-            <ArticleCard article={article} key={`${article.name}${index}`} />
-          );
+          const uniqueId = uuidv4();
+          return <ArticleCard article={article} />;
```

Comment:
Note how here you called the prop article, but single article in the other place you use it. Once you destructure the prop in article card, this will break